tools/ci_build/github/apple/objectivec/assemble_objc_pod_package.py

```diff
@@ -29,33 +30,66 @@
     "objectivec",
 ]
 
-# pod source files
-source_files = [
-    "objectivec/include/*.h",
-    "objectivec/*.h",
-    "objectivec/*.m",
-    "objectivec/*.mm",
-]
-
-# pod public header files
-# note: these are a subset of source_files
-public_header_files = [
-    "objectivec/include/*.h",
-]
-
-# pod test source files
-test_source_files = [
-    "objectivec/test/*.h",
-    "objectivec/test/*.m",
-    "objectivec/test/*.mm",
-]
+all_objc_files = {
+    "source_files": [
+        "objectivec/include/*.h",
+        "objectivec/*.h",
+        "objectivec/*.m",
+        "objectivec/*.mm",
+    ],
+    "public_header_files": [
+        "objectivec/include/*.h",
+    ],
+    "test_source_files": [
+        "objectivec/test/*.h",
+        "objectivec/test/*.m",
+        "objectivec/test/*.mm",
+    ],
+}
+
+training_only_objc_files = {
+    "source_files": [
+        "objectivec/include/onnxruntime_training.h",
+        "objectivec/include/ort_checkpoint.h",
+        "objectivec/include/ort_training_session.h",
+        "objectivec/ort_checkpoint.mm",
+        "objectivec/ort_checkpoint_internal.h",
+        "objectivec/ort_training_session_internal.h",
+        "objectivec/ort_training_session.mm",
+    ],
+    "public_header_files": [
+        "objectivec/include/ort_checkpoint.h",
+        "objectivec/include/ort_training_session.h",
+        "objectivec/include/onnxruntime_training.h",
+    ],
+    "test_source_files": [
+        "objectivec/test/ort_training_session_test.mm",
+        "objectivec/test/ort_checkpoint_test.mm",
+        "objectivec/test/ort_training_utils_test.mm",
+    ],
+}
 
 # pod test resource files
 test_resource_files = [
     "objectivec/test/testdata/*.ort",
+    "onnxruntime/test/testdata/training_api/*",
```

Comment:
is this one also training-only?

Reply:
yes, I moved it so it is only included when training is enabled